mws/resource_mws_pas.go

```diff
@@ -0,0 +1,112 @@
+package mws
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/internal"
+	"github.com/databrickslabs/databricks-terraform/internal/util"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// NewPASAPI creates VPCEndpointAPI instance from provider meta
+func NewPASAPI(ctx context.Context, m interface{}) PASAPI {
```

Comment:
rename to `NewPrivateAccessSettingsAPI` as well as other things :)

Reply:
All done